### PR TITLE
0.0.10

### PR DESCRIPTION
**Pull request by victoriabernard92**
0.0.9 didn't have a changelog since it was a quick publishing fix. This PR gets us ready for 0.0.10 by:

- adding 0.0.9 and 0.0.10 changelog
- cleaning up package.json

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
 # Changelog
 
+## 0.0.10
+
+- ### Features
+
+  - **Allow extensionless files to be served - [victoriabernard92], [cloudflare/wrangler/issues/980], [pull/73]**
```
Comment (EverlastingBugstopper) [inline CHANGELOG.md:7]: need to add links

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+
+  Prior to this PR, `getAssetFromKv` assumed extensionless requests (e.g. `/some-path`) would be set up to be served as the corresponding HTML file in storage (e.g. `some-path.html`).
+  This fix checks the `ASSET_MANIFEST` for the extensionless file name _before_ appending the HTML extension. If the extensionless file exists (e.g. `some-path` exists as a key in the ASSET_MANIFEST) then we serve that file first. If the extensionless file does not exist, then the behavior does not change (e.g. it still looks for `some-path.html`).
+
+- ### Fixes
+
+  - **Fix URL parsing in serveSinglePageApp - [signalnerve],[sgiacosa], [issue/72], [pull/82]**
```
Comment (EverlastingBugstopper) [inline CHANGELOG.md:14]: need to add links

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,30 @@
...
+
+  This fixes an issue in `serveSinglePageApp` where the request.url is used as a string to retrieve static content. For example,
+  if a query parameter was set, the URL lookup would break. This fix uses a parsed URL instead of the string and adjusts the README.
+
+## 0.0.9
+
+- ### Fixes
+  
+  - **Building and publishing to npm - [victoriabernard92], [pull/78], [pull/79]**
```
Comment (EverlastingBugstopper) [inline CHANGELOG.md:23]: were these two PRs? any issue?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,30 @@
...
+   
+    Publishing to npm started to break after we implemented in TypeScript because the `prepack` step 
+    was not set to build the JS files correctly
```
Comment (EverlastingBugstopper) [inline CHANGELOG.md:26]:
```suggestion
    Added a `prepack` step that builds JavaScript files from the TypeScript source. This fixes previously broken `npm` publishes.
```

File: CHANGELOG.md (hunk 3 above)
Comment (victoriabernard92) [reply CHANGELOG.md:23]: There was no issue

**Pull request merged**